src/cookieManager.ts

```diff
@@ -29,6 +29,28 @@ function enrichIdentity(identity: LegacySDKCookie, now: number) {
   };
 }
 
+function getCookie(cookieName: string) {
+  const docCookie = document.cookie;
+  if (docCookie) {
+    const payload = docCookie.split('; ').find((row) => row.startsWith(cookieName + '='));
+    if (payload) {
+      return decodeURIComponent(payload.split('=')[1]);
+    }
+  }
+}
+
+export function loadIdentityFromCookieNoLegacy(
+  cookieName: string
+): Identity | OptoutIdentity | null {
+  const payload = getCookie(cookieName);
+  if (payload) {
+    const result = JSON.parse(payload) as unknown;
+    if (isValidIdentity(result)) return result;
+    if (isOptoutIdentity(result)) return result;
+  }
+  return null;
+}
+
```

Comment:
Spoke with Lionell about making this a separate function from loadIdentityFromCookie in the CookieManager since we don't want them doing any setting without init, just getting